### ISSUE TITLE
CFG Holder automatisch Update Alert Message

### ISSUE THREAD

**Issue by Martin11180** [TasmoAdmin/TasmoAdmin#1214]
Hello
just testing tasmoadmin
I currently only have one module installed to test
For me, when I want to update my device, I get this message automatically
Attention: If you previously changed the CFG_HOLDER, it is now reset. This means that the default user_config.h will be loaded from the Tasmota GIT and your previous settings will be overwritten!
The CFG Holder is like the original setting, see device data
10:22:15.266 MQT: STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"Küche_Nachtlicht","OtaUrl":"http://ota.tasmota. com/tasmota/release/tasmota.bin.gz","RestartReason":"Software/System restart","Uptime":"0T00:00:34","StartupUTC":"2024-07-17T09:21:41 ","Sleep":50,"CfgHolder":4617,"BootCount":3,"BCResetTime":"2024-07-17T10:12:05","SaveCount":8,"SaveAddress":"F4000"} }
what is wrong or is it a BUG

**Comment by inverse** [comment]
That's just a warning that's displayed when you choose the automatic updates if you are coming from a custom build I believe.  Are you coming from a custom build?

If your [Tasmota upgrade flow](https://tasmota.github.io/docs/Upgrading/#upgrade-flow) is valid it should be fine.

**Comment by Martin11180** [comment]
yes before that was a custom build
I have now done an OTA update
with this file
http://ota.tasmota.com/tasmota/release/tasmota-minimal.bin.gz
http://ota.tasmota.com/tasmota/release/tasmota-DE.bin.gz
and then reset the device
And removing the device from TasmoAdmin and adding it back unfortunately still gives the message

**Comment by inverse** [comment]
That message will always be displayed as a warning whenever you do an automatic upgrade unfortunately. 

Perhaps we could provide a setting to permanently dismiss it in the settings.

**Comment by Martin11180** [comment]
Yes, it would be better because you think you have to do something
The CfgHolder can be queried via STATUS1 and compared with 4617

**Comment by inverse** [comment]
That sounds like a better suggestion! Thanks

**Comment by Martin11180** [comment]
Read STATUS1 when creating the module

**Comment by reloxx13** [comment]
Triage note: this remains valid. The current maintainer comments already confirm the warning is shown unconditionally today, and the suggested improvement of checking `Status1` / `CfgHolder` is still not implemented in the current codebase. Keeping this open with medium priority.
